test/extensions/filters/http/wasm/test_data/null_plugin/plugin.cc

```diff
@@ -23,6 +25,38 @@ void PluginRootContext::onTick() {
   if (WasmResult::Ok != proxy_get_current_time_nanoseconds(&t)) {
     logError(std::string("bad proxy_get_current_time_nanoseconds result"));
   }
+  {
+    std::string function = "declare_property";
+    envoy::source::extensions::common::wasm::DeclarePropertyArguments args;
+    args.set_name("structured_state");
+    args.set_type(envoy::source::extensions::common::wasm::WasmType::FlatBuffers);
+    args.set_span(envoy::source::extensions::common::wasm::LifeSpan::DownstreamConnection);
+    // simple table {i : int64}
```

Comment:
Add comment on how to regenerate the table.

Reply:
I created a flatbuffer table {i : int64}, and then just printed the schema. I did the same for the instance. I didn't want to take more bazel dependency on generating/saving the content but can do that if necessary. WDYT?

Reply:
Just a textual sketch of the process is sufficient such that it could be regenerated if required for some reason.

Reply:
Added instructions.